pipelines.adoc

```diff
@@ -221,16 +221,16 @@ vi default-collection-image-registry-secret.yaml
 oc apply -f default-collection-image-registry-secret.yaml
```

Comment:
Please update to oc apply -f default-stack-image-registry-secret.yaml

Reply:
fixed, along with several other instances nearby